test/remote/remote_gateway_options_test.rb

```diff
@@ -10,7 +10,7 @@ def test_successfully_get_options
     gateway_classes = @environment.gateway_options
     braintree = gateway_classes.select { |each| each.name == "Braintree" }.first
     assert_equal "http://www.braintreepaymentsolutions.com/", braintree.homepage
-    assert_equal %w(credit_card third_party_token apple_pay android_pay google_pay), braintree.payment_methods
```

Comment:
I'm fine with the change, but for my own learning, why change this from a `comparison` to a `negative union .empty` ?

Reply:
The previous version of the test case fails because we have added new payment methods. Rather than attempt to keep this test updated, I decided to test for the inclusion for known payment methods.